common/styles/scss/_navbar.scss

```diff
@@ -173,3 +173,7 @@ body > .container {
     overflow-x: hidden;
     overflow-y: auto;
 }
+
+.navbar-collapse {
+    max-height: 340px;
```

Comment:
Why this specific value? Is this based on RBK's navbar?

Reply:
This value is in bootstrap for navbar-collapse but it isn't getting applied so I have added it here

Reply:
Done